examples/example_trca.py

```diff
@@ -88,8 +88,17 @@
 # TRCA classification
 # -----------------------------------------------------------------------------
 # Estimate classification performance with a Leave-One-Block-Out
-# cross-validation approach
-trca = TRCA(fs, n_bands, is_ensemble)
+# cross-validation approach.
+
+# We use the filterbank specification described in [2]_.
+filterbank = [[[6, 90], [4, 100]],  # passband freqs, stopband freqs (Wp, Ws)
```

Comment:
Instead of a list of list of list, how about something simpler like a list of tuples? Of course, will be necessary to change also the bandpass function you defined.

i.e.

upFreqPb = 90 # upper frequency Passband
upFreqSb = 100 # upper frequency Stopband

lowFreqsPb = [6,14,22,30,38,46,54] # lower frequencies Passband
lowFreqsSb = [4,10,16,24,32,40,48] # lower frequencies Stopband

if(len(lowFreqsPb) == len(lowFreqsSb)):
    filterbank = [ (lowFreqsPb[i], upFreqPb,lowFreqsSb[i], upFreqSb) for i,e in enumerate(lowFreqsPb)] #list of tuples = [(low_freq_passband, high_freq_passband,low_freq_stopband, high_freq_stopband),...] 
else:
    print("lowFreqsPb and lowFreqsSb must have the same legnth")

Reply:
Actually I like keeping the `Wp`, `Ws` naming scheme because it is closer to the scipy.signal.cheb1ord(). 

So I have done this as a middle-ground : [(A, B), (C, D)], which can be instantiated pretty much like you want:

```python
i.e.

upFreqPb = 90 # upper frequency Passband
upFreqSb = 100 # upper frequency Stopband

lowFreqsPb = [6,14,22,30,38,46,54] # lower frequencies Passband
lowFreqsSb = [4,10,16,24,32,40,48] # lower frequencies Stopband

if(len(lowFreqsPb) == len(lowFreqsSb)):
    filterbank = [(p, upFreqPb), (s, upFreqSb) for p, s in zip(lowFreqsPb, lowFreqsSb)]
else:
    print("lowFreqsPb and lowFreqsSb must have the same legnth")